src/Lexers/ModelLexer.php

```diff
@@ -206,7 +207,7 @@ private function buildColumn(string $name, string $definition)
         $data_type = null;
         $modifiers = [];
 
-        $tokens = preg_split('#".*?"(*SKIP)(*FAIL)|\s+#', $definition);
+        $tokens = preg_split('#("|\').*?("|\')(*SKIP)(*FAIL)|\s+#', $definition);
```

Comment:
We would probably want to ensure the opening and closing quotes match. So maybe the backreference of `\1`?